src/player.cpp

```diff
@@ -10189,10 +10189,8 @@ int player::get_env_resist( body_part bp ) const
     }
 
     for( const bionic &bio : *my_bionics ) {
-        for( const auto &element : bio.info().env_protec ) {
-            if( element.first == bp || ( bp == bp_eyes && element.first == bp_head ) ) {
-                ret += element.second;
-            }
+        if( bio.info().env_protec.find( bp ) != bio.info().env_protec.cend() ) {
```

Comment:
```suggestion
        if( bio.info().env_protec.find( bp ) != bio.info().env_protec.end() ) {
```
Comparing to `cend` will work, but it's an unconventional choice.